cypress/integration/nx/general.spec.ts

```diff
@@ -9,4 +9,8 @@ describe('Default site', () => {
     cy.visit('//')
     cy.url().should('eq', `${Cypress.config().baseUrl}/`)
   })
-})
\ No newline at end of file
+
+  it('serves generated public files', () => {
+    cy.request(`service-worker.js`)
```

Comment:
You can be more explicit with the assetion here by making sure the status returned is a 200. Something like this:
```suggestion
    cy.request('service-worker.js').then((res) => {
      expect(res.status).to.eq(200)
    })
```

Reply:
👍  went ahead and added that and a check for the content-type (in case it's serving some fallback html instead of js!)